task.sql

```diff
@@ -1,33 +1,72 @@
+-- Drop the old database
+CREATE DATABASE ShopDB;
```

Comment:
You should include `DROP DATABASE ShopDB;` before this line to handle cases where the database already exists.